src/index.ts

```diff
@@ -5,8 +5,11 @@ import { router } from './routes';
 import { RegisterRoutes } from '../build/routes';
 import { ENV_VARIABLES } from './config/config';
 import dotenv from 'dotenv';
+import { errorHandler } from './middlewares/error-handler-middleware';
 const app = express();
 
+dotenv.config();
```

Comment:
I think we can replicate the same as the node template. I mean, we can create a method and wrap all these lines: cors, express, etc.

What do you think?

Reply:
Yes, make sense. 
Fixed.